### PR TITLE
Crowbar machines list cleanup

### PR DESCRIPTION
**Pull request by jdsn**
moves the call of 
`crowbar machines list | sort | grep | something`
to a function and replaces all usages of `crowbar machines list`.

This does not save code, but it becomes more readable.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -464,10 +464,29 @@ function get_disk_id_by_serial_and_libvirt_type()
     echo -n "$diskid"
 }
 
+function get_all_crowbar_nodes()
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: can be confused with get_crowbar_node

Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: renamed to `get_all_nodes`

**Comment by aspiers** [comment]
+1

**Comment by rsalevsky** [comment]
:+1:

**Pull request merged**